### PR TITLE
Fix Max lookback issue

### PR DESCRIPTION
**Pull request by areeda**
The --max-online-lookback argument did not work in all paths through the program. This is fixed and several logging messages add a human readable date/time to gps times.

For clarity the lookback is needed so that we we do not use the guardian channel in raw frames when processing long ti e intervals

### CODE REVIEW

**Comment by eagoetz** [comment]
@areeda Thanks for putting this together. It looks like there are some `flake8` failures to address, but I wanted to ask about the disabled test. Has it always been failing or does it fail now because of changes to this PR?

File: omicron/cli/archive.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
 # example channel indir: L1:SUS-PR3_M1_DAMP_T_IN1_DQ
 chpat = re.compile(".*/?([A-Z][1-2]):(.+)$")
 # example trigger file: L1-SUS_PR3_M1_DAMP_T_IN1_DQ_OMICRON-1336799058-8064.h5
-tfpat = re.compile("([A-Z][0-9])-(.+)-(\\d+)-(\\d+)\\.(.*)$")
+tfpat = re.compile("([A-Z][0-9])[-_:](.+)-(\\d+)-(\\d+)\\.(.*)$")
```
Comment (eagoetz) [inline omicron/cli/archive.py:65]: Can you double-check that this change is needed. Thank you!

File: omicron/cli/conda_run.py
Comment (areeda) [inline omicron/cli/conda_run.py:1]: Delete this? --

File: omicron/cli/process.py [hunk 2]
```diff
@@ -458,8 +495,15 @@ def create_parser():
 
 
 def main(args=None):
```
Comment (areeda) [inline omicron/cli/process.py:497]: Where we left off

File: omicron/cli/process.py [hunk 3]
```diff
@@ -977,17 +1045,21 @@ def main(args=None):
...
+    if len(segs) == 0:
+        logger.info('No analyzable segments found. Exiting.')
+        exit(0)
+    else:
+        logger.info("Final data segments selected as")
+        for seg in segs:
+            logger.info(f"    {gps_to_hr(seg[0])} {gps_to_hr(seg[1])} {abs(seg)}")
+        logger.info(f"Duration = {abs(segs)} seconds")
 
     span = type(trigsegs)([trigsegs.extent()])
```
Comment (areeda) [inline omicron/cli/process.py:1057]: Wat is this line doing?

Comment (areeda) [reply omicron/cli/process.py:1057]: and is frame avaiklabilityv abded with guarrdian/dqsedb segs?

File: omicron/cli/process.py [hunk 4]
```diff
@@ -1011,55 +1083,92 @@ def main(args=None):
     dag.set_dag_file(str(dagpath.with_suffix("")))
 
     # set up condor commands for all jobs
-    condorcmds = {
+    base_condorcmds = {
         "accounting_group": args.condor_accounting_group,
         "accounting_group_user": args.condor_accounting_group_user,
         "request_disk": args.condor_request_disk,
-        "request_memory": 1024,  # MB
+        "request_memory": '1024',
```
Comment (eagoetz) [inline omicron/cli/process.py:1090]:
```suggestion
        "request_memory": '1024',  # MB
```

File: omicron/cli/process.py [hunk 5]
```diff
@@ -1011,55 +1083,92 @@ def main(args=None):
...
+    ppjob.add_condor_cmd('my.InitialRequestMemory', f'{ppmem}')
     ppjob.add_condor_cmd('request_memory',
-                         f'ifthenelse(isUndefined(MemoryUsage), {ppmem}, int(3*MemoryUsage))')
-    ojob.add_condor_cmd('allowed_job_duration', 3 * 3600)
-    ppjob.add_condor_cmd('periodic_release',
-                         '(HoldReasonCode =?= 26 || HoldReasonCode =?= 34 '
-                         '|| HoldReasonCode =?= 46) && (JobStatus == 5)')
+                         f'ifthenelse(isUndefined(MemoryUsage), {ppmem}, int(1.5*MemoryUsage))')
+    ppjob.add_condor_cmd('periodic_release', '(HoldReasonCode =?= 26 || HoldReasonCode =?= 34) '
+                         '&& (JobStatus == 5) && (time() - EnteredCurrentStatus > 10)')
```
Comment (areeda) [inline omicron/cli/process.py:1167]: do we need delaay n ojobs?

File: omicron/tests/test_parameters.py
Comment (areeda) [inline omicron/tests/test_parameters.py:1]: check if this should be updated.

Comment (eagoetz) [reply omicron/tests/test_parameters.py:1]: Right, I recommend not disabling tests if at all possible. Thanks for taking some time to figure out how to update or what to do with this test

File: omicron/condor.py [hunk 6]
```diff
@@ -424,8 +424,7 @@ def find_dagman_id(group, classad="OmicronDAGMan", user=getuser(),
                    **constraints)
     clusterid = job['ClusterId']
     if get_job_status(job) >= 3:
-        raise RuntimeError("DAGMan cluster %d found, but in state %r"
-                           % JOB_STATUS[job['JobStatus']])
+        raise RuntimeError(f"DAGMan cluster {clusterid} found, but in state {JOB_STATUS[job['JobStatus']]}")
```
Comment (areeda) [inline omicron/condor.py:427]: does get_job_status return tuple or int

File: omicron/utils.py [hunk 7]
```diff
@@ -118,3 +120,30 @@ def astropy_config_path(parent, update_environ=True):
...
+    """
+    dt = from_gps(int(gps))
+    dt_str = dt.strftime('%x %X')
+    ret = f'{int(gps)} ({dt_str})'
+    return ret
+
+
+def deltat_to_hr(dt):
+    """
+    Convert a time in secos to human readable
```
Comment (eagoetz) [inline omicron/utils.py:139]:
```suggestion
    Convert a time in seconds to human readable
```

File: scripts/conda_run.sh
Comment (areeda) [inline scripts/conda_run.sh:1]: delete this

File: omicron/condor.py (hunk 6 above)
Comment (areeda) [reply omicron/condor.py:427]: Job['JobStatus'] is an integer. JOB_STATUS is an array that converts that integer to a string

File: omicron/cli/process.py (hunk 3 above)
Comment (areeda) [reply omicron/cli/process.py:1061]: These are not causing problems so let's delay addressing this

File: omicron/cli/process.py (hunk 4 above)
Comment (areeda) [reply omicron/cli/process.py:1094]: done

File: omicron/cli/archive.py (hunk 1 above)
Comment (areeda) [reply omicron/cli/archive.py:65]: not needed. reverted

File: omicron/tests/test_parameters.py
Comment (areeda) [reply omicron/tests/test_parameters.py:1]: Uncommented and it runs now??

**Comment by areeda** [comment]
Review issues resolved

**Pull request merged**